slack_bolt/adapter/aws_lambda/lambda_s3_oauth_flow.py

```diff
@@ -62,7 +62,9 @@ def __init__(
         # the settings may already have pre-defined authorize.
         # In this case, the /slack/events endpoint doesn't work along with the OAuth flow.
         settings.authorize = InstallationStoreAuthorize(
-            logger=logger, installation_store=settings.installation_store
+            logger=logger,
+            installation_store=settings.installation_store,
+            bot_only=settings.installation_store_bot_only,
```

Comment:
In `InstallationStore` class, `installation_store_` is redundant. So, I chose just `bot_only` for it.